src/st/no0/no0.h

```diff
@@ -51,22 +52,26 @@ extern u16 g_EInitCommon[];
 extern u16 g_EInitParticle[];
 
 // Axe knight
-extern u16 g_EInitAxeKnightAxe[];
+extern EInit g_EInitAxeKnightAxe;
 
 // Skeleton
-extern u16 g_EInitSkeleton[];
-extern u16 g_EInitSkeletonPieces[];
-extern u16 g_EInitSkeletonBone[];
+extern EInit g_EInitSkeleton;
+extern EInit g_EInitSkeletonPieces;
+extern EInit g_EInitSkeletonBone;
 
 // Ouija Table
-extern u16 g_EInitOuijaTable[];
-extern u16 g_EInitOuijaTableComponent[];
+extern EInit g_EInitOuijaTable;
+extern EInit g_EInitOuijaTableComponent;
 
 // Flea Man
-extern u16 g_EInitFleaMan[];
+extern EInit g_EInitFleaMan;
+
+// Ghost (enemy)
+extern EInit g_EInitGhostEnemy;
 
 // Clock room
-extern u16 g_Statues[];
+extern EInit g_Statues;
 
-// Clocks?
-extern u16 D_us_80180A88[];
+// Seems shared between a number of entities - ghost spawner,
+// func_us_801C2B24 (clocks?), func_us_801CC750
+extern EInit D_us_80180A88;
```

Comment:
Still don't have a great name for this. The other two functions aren't spawners so `g_EInitSpawner` doesn't make much sense, leaving as is for now.